docs/faq.mdx

```diff
@@ -0,0 +1,100 @@
+
+For FAQs specific to the execution modes, see the [Qiskit Runtime execution modes FAQ](/guides/execution-modes-faq) guide.
+
+## How do I cite Qiskit in my research?
+
+Cite Qiskit by using the included [BibTeX file](https://github.com/Qiskit/qiskit/blob/main/CITATION.bib).
+
+## How do I cite an IBM Quantum computer in my research?
+
+For research papers, we encourage authors to acknowledge IBM Quantum using:
+
+>    We acknowledge the use of IBM Quantum services for this work. The views expressed are those of the authors, and do not reflect the official policy or position of IBM or the IBM Quantum team.
+
+Paper references should be cited as follows:
+
+> IBM Quantum. https://<span>quantum.ibm.com</span>/, 2021
+
+Quantum processing units in the paper should be referenced by their unique name (i.e., `ibmq_vigo`) and optionally
+adding the version (i.e., `ibmq_vigo` v1.0.2).  We also encourage referencing the processor.
+For example:
+
+> In this paper we used `ibmq_vigo`, which is one of the IBM Quantum Canary processors.
+
+An example of citing an IBM Quantum program:
+
+> IBM Quantum (2022). Estimator primitive (Version x.y.z) [computer software]. https://<span>quantum.ibm.com</span>/
+
+## How do I cite IBM Quantum Composer in my research?
+
+An example of citing IBM Quantum Composer:
+
+> IBM Quantum Composer. 2023. url: https://<span>quantum.ibm.com/composer</span>
+
+## Why do I receive the error message "AttributeError: `QuantumCircuit` object has no attribute `save_state`" when using `save_*`method on a circuit?
+  
+The `save_*` instructions are part of Qiskit Aer project,
+a high performance simulator for quantum circuits. These instructions do not
+exist outside of Qiskit Aer and are added dynamically to the
+[QuantumCircuit](/api/qiskit/qiskit.circuit.QuantumCircuit#quantumcircuit-class) class by Qiskit Aer on import. If you would like to
+use these instructions you must first ensure that you have imported
+`qiskit_aer` in your program before trying to call these methods. You
+can refer to [qiskit_aer.library](https://qiskit.org/ecosystem/aer/apidocs/aer_library.html) for the details of these custom
+
+## Why do my results from real devices differ from my results from the simulator?
+
+The simulator models an ideal environment, without noise or decoherence. When jobs are run on the real devices, noise from the environment and decoherence cause the qubits to behave differently than in an ideal environment.
+
+## How do I delete my account and data?
+
+To delete your account, go to your [account page](https://quantum.ibm.com/account#delete-account). Scroll to the Delete Account section, and click *Delete your account and account data*. You will receive an email with a link to delete your account.
+
+## Questions about cloud simulator retirement
+
+To migrate from cloud simulators to local simulators, see the [migration guide](/migration-guides/local-simulators).
+
+### Why are the cloud simulators being retired?
```

Comment:
for this section on simulators maybe we can just share a link out to the migration guide rather than duplicating the info?